### ISSUE TITLE
Design Intent - Accessibility Feedback - govcio-vebt, 35% Exemption Request from 85/15 Reporting Requirement Application (VA Form 22-10216), VA Form 22-10216

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#93661]
## Next Steps for the VFS team

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

- Sorry for the delay! I had everything written up but never actually pressed the button to create the issue. 🤦 
- In the meeting, I mentioned that I might provide coding notes on the [calculated VA beneficiary students percentage](https://www.figma.com/design/69FpDTQN5UErSV27txIKFh/VEBT-Form-22-10216%3A-35%25-Exemption?node-id=1-44347&t=XbpP0hUc1Nf0mCB1-4). I want to have a few conversations with other folks in the accessibility community of practice before I give any recommendations there, so I'll save that for your Midpoint Review!

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [ ] **Should:** When an [additional form is needed and users are notified mid-flow](https://www.figma.com/design/69FpDTQN5UErSV27txIKFh/VEBT-Form-22-10216%3A-35%25-Exemption?node-id=141-7043&t=XbpP0hUc1Nf0mCB1-4), screen reader working through the form by tabbing through interactive elements (or in forms mode for JAWS) aren't likely to realize the "Additional form needed" alert is there and are more likely to skip past it. I'm not hugely concerned about this since a comparable warning is provided at the end of the flow. But if it's important to notify users at this point, I'd encourage you to move that warning to a new page that follows after pressing Continue rather than conditionally revealing the alert on this page.
- [ ] **Should:** Wholeheartedly echo the feedback given by Erin and Allison to move the "How to submit" content to its own page after the review page.
- [ ] **Must:** I don't remember this coming up in the meeting, but from the Figma files it looks as though the sections on the [review page](https://www.figma.com/design/69FpDTQN5UErSV27txIKFh/VEBT-Form-22-10216%3A-35%25-Exemption?node-id=274-11315&t=XbpP0hUc1Nf0mCB1-4) are intended to be collapsible, with behavior similar to accordions. Definitely don't do that --- or, rather, if it's important for those sections to be collapsible, use the accordion component and the standard review page that comes out of the box with the forms library rather than making the new-style review headings also function as accordion-like triggers.
- [ ] **Implementation notes/reminders:** As with 22-10215, make sure the PDF that's generated is itself accessible, and I have concerns about potential accessibility barriers on the upload page.

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [x] Assign this ticket to the VFS team member that opened the Slack request
- [x] Add the VFS team product label
- [x] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [x] Add the Collaboration Cycle initiative milestone

**Comment by arianaadili** [comment]
These points have been addressed in our latest wireframes and user flows. Updates include adjusting how the "Additional form needed" warning appears to improve accessibility, ensuring the "How to submit" content is on its own page after the review page, and maintaining consistency with the standard review page design rather than introducing new collapsible behaviors. Additionally, we've taken accessibility into account for the generated PDF and upload page.
